cpp/tests/io/nested_json_test.cu

```diff
@@ -14,17 +14,121 @@
  * limitations under the License.
  */
 
+#include "cudf_test/column_wrapper.hpp"
+
 #include <io/json/nested_json.hpp>
 #include <io/utilities/hostdevice_vector.hpp>
 
+#include <cudf/io/datasource.hpp>
+#include <cudf/io/parquet.hpp>
+
 #include <cudf_test/base_fixture.hpp>
+#include <cudf_test/column_utilities.hpp>
 #include <cudf_test/cudf_gtest.hpp>
+#include <cudf_test/io_metadata_utilities.hpp>
+#include <cudf_test/table_utilities.hpp>
+
+#include <cudf/lists/lists_column_view.hpp>
 
 #include <rmm/cuda_stream.hpp>
 #include <rmm/cuda_stream_view.hpp>
 
+#include <string>
+
 namespace cuio_json = cudf::io::json;
 
+namespace {
+// Forward declaration
+void print_column(std::string const& input,
```

Comment:
These can probably be moved to a common location, but this can be a separate PR.

Reply:
That's just a forward declaration for usage by the functions between this forward declaration and the actual definition a few lines further down. Don't think there's need to to extract this to a common header.